Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoadTestDataDocumentBean.java

```diff
@@ -70,56 +74,64 @@
     private static final String GROWL_MESSAGE = "msgForGrowl";
 
     private String dialogTitle;
-    private DocumentMetadata withDocument;
-    private EventCode withEventCode;
+    private Document withDocument;
+    private EventCodeType withEventCode;
 
-    private DocumentMetadata selectedDocument;
-    private EventCode selectedEventCode;
+    private Document selectedDocument;
+    private EventCodeType selectedEventCode;
 
-    @Autowired
-    private LoadTestDataService loadTestDataService;
+    private LoadTestDataWSService wsService = new LoadTestDataWSServiceImpl();
     private Map<String, String> listStatusType;
+    private List<Document> documentList;
+    Map<Long, Patient> lookupPatient;
 
     public LoadTestDataDocumentBean() {
         selectedDocument = null;
     }
 
     // selected-record
-    public DocumentMetadata getSelectedDocument() {
```

Comment:
Curious, why was this switched to "Document"? With the new Document Data Submission service, we can upload a document meta data without uploading a document - this leads me to believe we may introduce errors in this case.

Reply:
the document is intended to handle some of the issue that came up during the adminGUI interacting with the model; xml-date > Date > string -- issue with the converter and additional properties that is only used by the adminGUI